CHANGELOG.md

```diff
@@ -21,6 +22,7 @@
 * [BUGFIX] Fix issue where all incoming HTTP requests have duplicate trace spans. #6920
 * [BUGFIX] Querier: do not retry requests to store-gateway when a query gets canceled. #6934
 * [BUGFIX] Querier: return 499 status code instead of 500 when a request to remote read endpoint gets canceled. #6934
+* [BUGFIX] Query-frontend: Validate tenant IDs according to documented behavior even when tenant federation is not enabled. #6959
```

Comment:
can you add a link to the documented behavior or explain which tenant IDs will now be rejected?

Reply:
Added a link to the Mimir docs.